web-portlet/src/java/org/sakaiproject/portlets/PortletIFrame.java

```diff
@@ -904,7 +912,18 @@ else if (newPageTitle.length() > MAX_TITLE_LENGTH)
             {
                 if ((infoUrl != null) && (infoUrl.length() > 0) && (!infoUrl.startsWith("/")) && (infoUrl.indexOf("://") == -1))
                 {
-                    infoUrl = "http://" + infoUrl;
+                    infoUrl = "//" + infoUrl;
```

Comment:
there is a 'validateUrl' method called when infourl is edited which generates error for this www.university.edu/course/STAT-101 url but if 'http://www.university.edu/course/STAT-101' this will get added. So we should not let user add such url?